### DISCUSSION TITLE
hsrb_kinematics package for noetic

### DISCUSSION THREAD

**Discussion by Janillaa** [HumanSupportRobot/QandA#51] [Q&A] [answered]
Hi there, 

The hsrb_moveit_plugins package has a hsrb_moveit_kinematics plugin. The current release of this package is in foxy. We were wondering if/where this package can be downloaded in noetic? We are currently running software release 22.10.

Thanks!

**Comment by hsr-admin** [comment, marked as answer]
The source code that works with noetic can be found at
https://git.hsr.io/tmc/hsrb_moveit/-/tree/v0.6.3-community/hsrb_moveit_plugins/hsrb_moveit_kinematics


It can also be found at the following link:
https://docs.hsr.io/hsrio_manual_en/gitlab/gitlab.html#id2
